src/components/LocationPage/LocationPageHeading.tsx

```diff
@@ -10,10 +11,21 @@ const LocationPageHeading: React.FC<{ region: Region; isEmbed: boolean }> = ({
   } else if (region instanceof County) {
     return (
       <Fragment>
```

Comment:
A bit of a orthogonal comment, but what are your thoughts on having:

```jsx
<HeaderTitle isEmbed={isEmbed}>
  <LocationPageHeading region={region} isEmbed={isEmbed} />
</HeaderTitle>
```
vs 
```
<LocationPageHeading region={region} isEmbed={isEmbed} />
```
where the `HeaderTitle` is basically in the `<LocationPageHeading />`.

Reply:
good point!